ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/service/ViewSpecificationConverter.java

```diff
@@ -16,7 +16,7 @@
 import java.util.Map;
 import java.util.function.Predicate;
 
-import static be.vlaanderen.informatievlaanderen.ldes.server.admin.rest.constants.ViewSpecificationConverterConstants.*;
```

Comment:
Since, you are no longer using these constants, can you delete them in the class ViewSpecificationConverterConstants?

Reply:
Yes it could